### ISSUE TITLE
Cannot render Stand-alone SKU Backend View

### ISSUE THREAD

**Issue by dneckel** [KenticoDevTrev/DynamicRouting#17]
If I enable the setting "Allow stand-alone SKUs" and try to access the view via "Products" --> "Stand-alone SKUs" I get the following Internal Server Error.

```
Message: Exception of type 'System.Web.HttpUnhandledException' was thrown.

Exception type: System.Web.HttpUnhandledException
Stack trace:
at System.Web.UI.Page.HandleError(Exception e)
at System.Web.UI.Page.ProcessRequestMain(Boolean includeStagesBeforeAsyncPoint, Boolean includeStagesAfterAsyncPoint)
at System.Web.UI.Page.ProcessRequest(Boolean includeStagesBeforeAsyncPoint, Boolean includeStagesAfterAsyncPoint)
at System.Web.UI.Page.ProcessRequest()
at System.Web.UI.Page.ProcessRequest(HttpContext context)
at ASP.cmsmodules_ecommerce_pages_tools_products_products_frameset_aspx.ProcessRequest(HttpContext context)
at System.Web.HttpApplication.CallHandlerExecutionStep.System.Web.HttpApplication.IExecutionStep.Execute()
at System.Web.HttpApplication.<>c__DisplayClass285_0.b__0()
at System.Web.HttpApplication.ExecuteStepImpl(IExecutionStep step)
at System.Web.HttpApplication.ExecuteStep(IExecutionStep step, Boolean& completedSynchronously)

Message: Object reference not set to an instance of an object.

Exception type: System.NullReferenceException
Stack trace:
at DynamicRouting.Kentico.Base.DynamicRoutingDocumentURLProvider.GetPresentationUrlInternal(TreeNode node, String preferredDomainName)
at CMS.UIControls.ContentUrlRetriever.GetRequestedUrlInternal(UIPageURLSettings settings, Boolean ajaxRequest)
at CMS.UIControls.ContentUrlRetriever.GetRequestedUrl(String eventArgument, Boolean ajaxRequest)
at CMS.UIControls.ContentUrlRetriever.OnPreInit(Object sender, EventArgs e)
at System.EventHandler.Invoke(Object sender, EventArgs e)
at System.Web.UI.Page.OnPreInit(EventArgs e)
at CMS.UIControls.AbstractCMSPage.OnPreInit(EventArgs e)
at CMS.UIControls.CMSPage.OnPreInit(EventArgs e)
at CMS.Ecommerce.Web.UI.CMSProductsPage.OnPreInit(EventArgs e)
at CMSModules_Ecommerce_Pages_Tools_Products_Products_Frameset.OnPreInit(EventArgs e) in C:\inetpub\wwwroot\jb_shop\CMS\CMSModules\Ecommerce\Pages\Tools\Products\Products_Frameset.aspx.cs:line 196
at System.Web.UI.Page.PerformPreInit()
at System.Web.UI.Page.ProcessRequestMain(Boolean includeStagesBeforeAsyncPoint, Boolean includeStagesAfterAsyncPoint)
```

**Comment by KenticoDevTrev** [comment]
I'm a little confused, a stand alone sku isn't a page, so I'm not sure how it would render. Dynamic routing depends on having a page with a relative url to match. Can you put the dynamic route attribute and some more info?

**Comment by KenticoDevTrev** [comment]
Or is this an error in the CMS admin?

**Comment by dneckel** [comment]
Yes this is on the CMS Admin (Mother).
E-Commerce --> Products --> Stand-alone SKUs

**Comment by KenticoDevTrev** [comment]
Got it, I'll get this checked and fixed up right away.

**Comment by KenticoDevTrev** [comment]
This has been resolved in commit 2e535e819f4c5528916bf286cecca5ca41d448a0 and nuget package DynamicRouting.Kentico 12.29.9 and DynamicRouting.Kentico.MVC 12.29.9

Thanks for spotting this!
